docs/content.zh/docs/connectors/pipeline-connectors/overview.md

```diff
@@ -30,24 +30,26 @@ Flink CDC 提供了可用于 YAML 作业的 Pipeline Source 和 Sink 连接器
-| <font color="DarkCyan">3.0.x</font> |                        <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>                        |      Mysql      |              StarRocks,Doris               |                                   |
-| <font color="DarkCyan">3.1.x</font> | <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>, <font color="MediumVioletRed">1.19.\*</font> |      Mysql      |        StarRocks,Doris,Paimon,Kafka        | 仅 flink-cdc 3.1.1 支持 flink 1.19.* |
-| <font color="DarkCyan">3.2.x</font> | <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>, <font color="MediumVioletRed">1.19.\*</font> |      Mysql      | StarRocks,Doris,Paimon,Kafka,ElasticSearch |                                   |
+|    Flink<sup>®</sup> CDC Version    |                                                                               Flink<sup>®</sup> Version                                                                                | Pipeline Source |                          Pipeline Sink                          | 备注                                |
+|:-----------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------:|:---------------------------------------------------------------:|:----------------------------------|
+| <font color="DarkCyan">3.0.x</font> |                                               <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>                                               |      Mysql      |                         StarRocks,Doris                         |                                   |
+| <font color="DarkCyan">3.1.x</font> |                        <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>, <font color="MediumVioletRed">1.19.\*</font>                        |      Mysql      |                  StarRocks,Doris,Paimon,Kafka                   | 仅 flink-cdc 3.1.1 支持 flink 1.19.* |
+| <font color="DarkCyan">3.2.x</font> |                        <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>, <font color="MediumVioletRed">1.19.\*</font>                        |      Mysql      |           StarRocks,Doris,Paimon,Kafka,ElasticSearch            |                                   |
+| <font color="DarkCyan">3.3.x</font> | <font color="MediumVioletRed">1.17.\*</font>, <font color="MediumVioletRed">1.18.\*</font>, <font color="MediumVioletRed">1.19.\*</font>, <font color="MediumVioletRed">1.20.\*</font> |      Mysql      | StarRocks,Doris,Paimon,Kafka,ElasticSearch,Oceanbase,Maxcompute |                                   |
```

Comment:
Seems 3.3.x have dropped Flink 1.18 and earlier support?

Reply:
Removed.